docs/report.md

```diff
@@ -20,8 +20,31 @@ The Model analyzer can run inferences with models across multiple parameters. It
 
 ## Summary Reports
 
-The most basic type of report is the summary report which the Model Analyzer generates by default for each model. The output directory will, by default, contain 3 subdirectories. The summary report for a model will be located in `[export-path]/reports/<model name>`. The report should look something like the one shown here.
+The most basic type of report is the *Summary Report* which the Model Analyzer's `analyze` subcommand generates by default for each model. After profiling, users can run the `analyze` subcommand on the direcotry where the data from the profiling is stored as a checkpoint.
```

Comment:
I think the second sentence is not required. They can infer this from the example below.